composer.lock

```diff
@@ -0,0 +1,3256 @@
+{
```

Comment:
composer.lock should not be committed by default. 

removed it: dd59d75f842d012168fd32af465a26ac8cb4300b